artifactory/commands/terraform/terraformpublish.go

```diff
@@ -316,8 +328,7 @@ func (tpc *TerraformPublishCommand) getPublishTarget(moduleName string) string {
 	return path.Join(tpc.repo, tpc.namespace, moduleName, tpc.provider, tpc.tag+".zip")
 }
 
-// We identify a Terraform module by the existence of a file with a ".tf" extension inside the module directory.
-// isTerraformModule search for a file with a ".tf" extension inside and returns true it founds at least one.
+// We identify a Terraform module by the existence of at least one file with a ".tf" extension inside the module directory.
```

Comment:
We identify a Terraform module by the existence of at least one file with a ".tf" extension inside the module directory.
--> 
We identify a Terraform module by having at least one file with a ".tf" extension inside the module directory.